projects/openssh/build.sh

```diff
@@ -30,8 +33,12 @@ $CXX $CXXFLAGS -std=c++11 -I. -L. -Lopenbsd-compat -g \
 $CXX $CXXFLAGS -std=c++11 -I. -L. -Lopenbsd-compat -g \
 	regress/misc/fuzz-harness/sig_fuzz.cc -o $OUT/sig_fuzz \
 	-lssh -lopenbsd-compat $STATIC_CRYPTO -lFuzzingEngine
+$CXX $CXXFLAGS -std=c++11 -I. -L. -Lopenbsd-compat -g \
```

Comment:
Would it make sense to generalize these via simply building a fuzzer with: `for fuzz in regress/misc/fuzz-harness/*_fuzz.cc`? This will mean you don't need to send PRs to this repo any time you add a new fuzzer.

Reply:
That would be nice, but in practice they will require different object files be linked (e.g. the auth-options.o in the above)

Reply:
Maybe a Makefile.oss-fuzz would do the trick - do you have any objection to that approach?

Reply:
A Makefile.oss-fuzz sounds good for a future PR :)